src/sentry/api/event_search.py

```diff
@@ -1805,30 +1805,48 @@ def validate_argument_count(self, field, arguments):
         Function(
             "percentile_range",
             required_args=[
-                DurationColumn("column"),
+                DurationColumnNoLookup("column"),
                 NumberRange("percentile", 0, 1),
                 DateArg("start"),
                 DateArg("end"),
                 NumberRange("index", 1, None),
             ],
             aggregate=[
-                u"quantileIf({percentile:.2f})({column},and(greaterOrEquals(timestamp,toDateTime('{start}')),less(timestamp,toDateTime('{end}'))))",
-                None,
+                u"quantileIf({percentile:.2f})",
+                [
+                    ArgValue("column"),
+                    [
+                        "and",
+                        [
+                            ["lessOrEquals", [["toDateTime", [ArgValue("start")]], "timestamp"]],
+                            ["greater", [["toDateTime", [ArgValue("end")]], "timestamp"]],
```

Comment:
Did these invert to appease the snuba query syntax? Flipping the function names doesn't make sense to me as the range is now 'less than or equals to start, and greater than the end' which seems like it is not counting the selected window.

Reply:
Unfortunately, yes. If the condition was written as `["greaterOrEquals", ["timestamp", ["toDateTime", [ArgValue("start")]]]]`, snuba special cases this as `greaterOrEquals(timestamp(toDatetime(start)))` which is not what we want, and inverting these conditions allows us to specify them in a way snuba expects.

Reply:
Could we add a comment explaining this?

Reply:
Added comments in 2f813b4